Jenkinsfile

```diff
@@ -0,0 +1,124 @@
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the GOUpdate step of Reactome's release.
+// This step synchronizes Reactome's GO terms with Gene Ontology. 
+// It requires that the ConfirmReleaseConfigs step has been run successfully before it can be run.
+def currentRelease
+def previousRelease
+pipeline {
+	agent any
+
+	stages {
+		// This stage checks that an upstream project, ConfirmReleaseConfig, was run successfully for its last build.
```

Comment:
If you do decide to quote step names, you could quote "ConfimReleaseConfig" on line 4 and line 11 for consistency.

Also (nitpickish) I notice that on line 4 "ConfirmReleaseConfig" has an 's' at the end but it doesn't on line 11.